src/ocean/io/serialize/StringStructSerializer.d

```diff
@@ -219,7 +220,7 @@ public class StringStructSerializer ( Char )
 
     public void open ( ref Char[] output, cstring name )
     {
-        assert(this.indent.length == 0, "Non-zero indentation in open");
+        verify(this.indent.length == 0, "Non-zero indentation in open");
```

Comment:
1) Error message is reverse wrong btw.
2) I think all those check must be `assert` (probably with reusable exception even) as serialization often happens with I/O data

Reply:
It doesn't look that the error message is wrong :)

Reply:
So many negatives :grumpy: